src/main/resources/templates/macros/report/docstring.vm

```diff
@@ -8,7 +8,7 @@
           <a>Doc string</a>
       </div>
       <div id="msg-$docStringId" class="collapse collapsable-details">
-        <pre>$docString.getValue()</pre>
+        <pre>$escHtml.translate($docString.getValue())</pre>
```

Comment:
If a developer is writing a bdd specification thinking how it will render in an html report, in my opinion is a bad specification. Anyway, what do you propose? We can set a flag in the configuration object to escape the docStrings.